content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,55 @@
+# Add Dependencies or Outputs to a Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of its data requirements are missing from `dvc.yaml`. Namely, one of:
+
+- Files or directories in the <abbr>workspace</abbr> that are dependencies of
+  the stage are missing from `deps`.
+
+- Output files or directories that the stage creates, which are already in the
+  workspace are missing from `outs`.
+
+Follow the steps below to add existing files/directories as
+<abbr>dependencies</abbr> or <abbr>outputs</abbr> to a stage without executing
+it again, which can be expensive/time-consuming, and is unnecessary.
+
+We start with an example stage `prepare`, which has a single dependency and
+output. To add a missing dependency `data/raw.csv` and output `data/validate` to
```

Comment:
Let's be clearer, since this part got complicated:
```suggestion
output. To add a missing dependency (`data/raw.csv`) as well as a missing output (`data/validate`) to
```

Reply:
Done